Added PCI DSS docs for logs (#15702)
DataDog/documentation: content/en/data_security/logs.md

* Security rules cannot include message template variables.
* Security rules cannot be notified by webhooks.

> If you have any questions about how the Log Management Service satisfies the applicable requirements under HIPAA, contact your account manager. Previously, HIPAA-enabled customers needed to use specific endpoints to submit logs in order to enforce specific encryptions. This is no longer necessary. The encryptions are enabled on all log submission endpoints.

Reviewer: ```suggestion
If you have any questions about how the Log Management Service satisfies the applicable requirements under HIPAA, contact your account manager. HIPAA-enabled customers do not need to use specific endpoints to submit logs to enforce specific encryptions. The encryptions are enabled on all log submission endpoints.
```

## PCI DSS compliant Datadog Orgs

> Datadog allows customers to send logs to PCI DSS compliant Datadog org upon request. To setup a PCI compliant Datadog Org, please follow these steps. 

Reviewer: Thanks for clarifying! Changed it back.

1. Setup a New Datadog Org in US1. Currently we only offer PCI complaince for new orgs in Datadog US1 region.

> 2. Contact Datadog support or Customer Success and request that the new org be configured as PCI compliant org.

Reviewer: ```suggestion
2. Contact Datadog support or Customer Success to request that the new org be configured as a PCI-compliant org.
```

> 3. Enable Datadog Audit Trail in the new org. Datadog Audit Trail is a requirement to remain compliant with PCI DSS.

Reviewer: ```suggestion
3. Enable Audit Trail in the new org. Audit Trail must be enabled and remain enabled for PCI DSS compliance.
```

> 4. Datadog Support or Customer Success confirms that the new org is now PCI DSS compliant

Reviewer: ```suggestion
4. Datadog support or Customer Success confirms that the new org is PCI DSS compliant.
```

> 5. Update the Datadog agents to ship logs to the following dedicated PCI compliant endpoint 

Reviewer: ```suggestion
5. Update the Datadog Agents to ship logs to the following dedicated PCI-compliant endpoint: 
```

    `agent-http-intake-pci.logs.datadoghq.com`
  

> If you have any questions about how the Log Management Service satisfies the applicable requirements under PCI DSS, contact your account manager.

Reviewer: ```suggestion
If you have any questions about how the Log Management service satisfies the applicable requirements under PCI DSS, contact your account manager.
```

**Note:**

> Logs can be viewed through various pages in the Datadog platform. All manifestations of logs in the Datadog platform, including pages such as Traces are part of the log management product.

Reviewer: ```suggestion
Logs can be viewed in various Datadog products. All logs viewed in the Datadog UI, including logs viewed in APM trace pages, are part of the Log Management product.
```
